lib/ansible/modules/source_control/git.py

```diff
@@ -169,6 +169,15 @@
               can be separated from working tree.
         version_added: "2.7"
 
+    gpg_whitelist:
+        description:
+            - A list of trusted GPG fingerprints which if verify_commit is set
```

Comment:
@acozine could you please take a look at the wording here?

Reply:
I'd change this description to:
```
  - A list of trusted GPG fingerprints to compare to the fingerprint of the GPG-signed commit. 
  - Only used when I(verify_commit=yes).
```

Reply:
Applied